packages/babel-parser/src/parse-error/standard-errors.ts

```diff
@@ -117,6 +117,8 @@ export default {
   ImportCallSpreadArgument: "`...` is not allowed in `import()`.",
   ImportJSONBindingNotDefault:
     "A JSON module can only be imported with `default`.",
+  ImportReflectionNotBinding:
+    'Only `import module x from "./module"` is supported.',
```

Comment:
Nit:
```suggestion
    'Only `import module x from "./module"` is valid.',
```
otherwise it reads as if it's just something that _Babel_ does not support.